Changelog.md

```diff
@@ -0,0 +1,2 @@
+### v1.4.0 / 2019-05-03
+[Full Changelog](https://github.com/wundertax/rubocop-wundertax/pull/7)
```

Comment:
we normally follow the convention of using `compare` from github:
```md
[Full Changelog](http://github.com/wundertax/rubocop-wundertax/compare/1.3.0...1.4.0)
```

check if that works otherwise leave it as it is